examples/Server.java

```diff
@@ -592,8 +592,12 @@ public void run(String[] args) {
         } catch (IOException e) {
             e.printStackTrace();
         } finally {
-            if (serverSocket != null) {
-                serverSocket.close();
+            try {
+                if (serverSocket != null) {
+                    serverSocket.close();
+                }
+            } catch (Exception e) {
```

Comment:
I think this should be `IOException` instead of generic `Exception`.  Does that line up with the exception error you were seeing on this?

Reply:
Yes, that seems more appropriate and works.